requirements-dev.txt

```diff
@@ -4,7 +4,7 @@ certifi==2018.1.18
 chardet==3.0.4
 flake8==3.5.0
 idna==2.6
-lark-parser==0.5.3
+lark-parser==0.6.4
```

Comment:
It's totally relevent. However, I think it isn't related to this PR in particular, isn't?

Reply:
It's not related. But some tests fail when using lark-parser==0.5.3. So I suppose 0.6 is now required ?

Reply:
Oh sorry, you're right. The 0.6.0 of Lark broke backward compatibility on some points, I forgot this point.